tests/test_wallet.py

```diff
@@ -723,11 +766,16 @@ def test_sign_external_psbt(node_factory, bitcoind, chainparams):
     l1.rpc.signpsbt(psbt)
 
 
-@unittest.skipIf(TEST_NETWORK == 'liquid-regtest', '')
 def test_sign_and_send_psbt(node_factory, bitcoind, chainparams):
     """
     Tests for the sign + send psbt RPCs
     """
+    # CLN returns PSBTv0 and PSETv2, for now
+    is_psbt_v2 = chainparams['elements']
+    if is_psbt_v2:
+        # FIXME Elements Core(And Bitcoin Core with PSBTv2) cannot use joinpsbt call yet
+        return
```

Comment:
I think there's a `unittest.skip()` or something for this?

Reply:
added